cromwell-helm/values.yaml

```diff
@@ -2,6 +2,12 @@ service:
   type: ClusterIP
   port: 8000
 
+cromwellConfig:
```

Comment:
I think I would simplify these as suggested below. Since you access them via the full path, "config" doesn't need to be repeated. And it seems like the general pattern is to minimize camel-cased value names.

```
config:
  dir:
  file:
  gcsBucket:
  gcsProject: